cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -94,6 +94,7 @@ func (s *Server) GetAvailablePackages(ctx context.Context, request *corev1.GetAv
 		name, found, err := unstructured.NestedString(repoUnstructured.Object, "metadata", "name")
 		if err != nil || !found {
 			log.Errorf("required field metadata.name not found on HelmRepository: %w:\n%v", err, repoUnstructured.Object)
+			// just skip over to the next one
```

Comment:
Isn't an error here a programming error - in that we expect it not to be possible, but if it is, it's because of an incorrect assumption on our part (ie. than metadata.name should always exist) ? I'm always a bit wary of ignoring any error (from a functionality pov). But hey, it's a plugin and you're the author :P. As you say, it'll make no difference anyway is there will always be a name.

Reply:
Not a fan of ignoring errors either. But I am not doing that here. Logging an error isn't the same as ignoring it.

Reply:
Yeah, that's what I meant by "from a functionality pov" - I realise you're logging the error. Fine either way.